sources/packages/backend/apps/api/src/route-controllers/user/user.aest.controller.ts

```diff
@@ -0,0 +1,49 @@
+import { Controller, Get, Put } from "@nestjs/common";
+import { UserService } from "../../services";
+import BaseController from "../BaseController";
+import { UserToken } from "../../auth/decorators/userToken.decorator";
+import { IUserToken } from "../../auth/userToken.interface";
+import { BCeIDAccountsAPIOutDTO } from "./models/bceid-accounts.dto";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { AllowAuthorizedParty, Groups } from "../../auth/decorators";
+import { ApiTags, ApiUnprocessableEntityResponse } from "@nestjs/swagger";
+import { UserControllerService } from "..";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Controller("users")
+@ApiTags("users")
+export class UserAESTController extends BaseController {
+  constructor(
+    private readonly service: UserService,
+    private readonly userControllerService: UserControllerService,
+  ) {
+    super();
+  }
+
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Not able to retrieve BCeID business account details for the current authenticated user.",
+  })
+  @Get("bceid-accounts")
+  async getAllBCeIDs(
+    @UserToken() userToken: IUserToken,
+  ): Promise<BCeIDAccountsAPIOutDTO> {
+    return await this.userControllerService.getAllBCeIDs(userToken);
+  }
+
+  /**
+   * Creates or updates Ministry user information.
+   * @param userToken user token information to be updated.
+   */
+  @Groups(UserGroups.AESTUser)
+  @Put("aest")
```

Comment:
I believe that we need just the HTTP verb here as `@Put("aest")`.

Reply:
Do you mean what ann commented on https://github.com/bcgov/SIMS/pull/1627#pullrequestreview-1228384600 ?

Reply:
I am not sure. My point was just about removing the "aest" from the `@Put("aest")`. The "aest" was there before because the controller was shared. Now the entire controller is for "aest". We can have a chat if it is not clear.